src/test/java/net/phptravels/WebLogin.java

```diff
@@ -13,50 +13,39 @@
 public class WebLogin implements IAbstractTest {
 
     @Test()
-    @MethodOwner(owner = "qpsdemo")
-    @TestPriority(Priority.P3)
-    @TestLabel(name = "feature", value = {"web", "regression"})
     public void testHotelsFillForm() {
         HomePage homePage = new HomePage(getDriver());
         homePage.open();
         Assert.assertTrue(homePage.isPageOpened(), "Home page is not opened");
-        homePage = new HomePage(getDriver());
-        homePage.hotelFillForm("Coffee",  "24-04-2022", "25-04-2022", "5", "3", "1");
+        HotelsSearchResultPage hotelSearch = homePage.hotelFillForm("Coffee",  "24-04-2022", "25-04-2022", "5", "3", "1");
```

Comment:
`hotelSearch` is not good way to name pages/components in the tests
`hotelsSearchResultPage` is more clear. 
pages should end with Page etc
Check all other tests and rename please